### DISCUSSION TITLE
Support of Unstoppable Domains Connector

### DISCUSSION THREAD

**Discussion by fnnhnsn** [wevm/wagmi#1531] [Ideas]
Dear wagmi team,

We are big fans of your work here at Polychain Monsters and would like to add another connector. We would like to add support for Unstoppable Domains. It was already proposed here, but with no clear intentions to build it on their own: [https://github.com/wagmi-dev/wagmi/discussions/796](https://github.com/wagmi-dev/wagmi/discussions/796)
Of course, if that is not planned or helpful for you, we could just build a custom solution for ourselves. However, we thought it might be helpful to build something that could be reused by all wagmi users. 
What do you think? 
Thanks a lot in advance, and greetings from Germany!

**Comment by fenil233** [comment]
any idea on how to implement custom solution ?

**Reply by fnnhnsn** [reply]
> any idea on how to implement custom solution ?

Hey, we brainstormed today and will probably start with a custom open-source solution, that will be similar to the existing unstoppable domain packages here: https://github.com/unstoppabledomains/uauth/tree/main/packages
By that, we would not cause any package dependencies within wagmi and as we are currently in contact with the unstoppable domains team as well, we could hopefully get something done quickly.
Once we got a solution ready we would share it with you and the unstoppable domains team so that everyone can check whether it might make sense to integrate it some place somehow. :)
